### PR TITLE
[ApiBundle][Address] Add address log entries operation

### PR DESCRIPTION
**Pull request by Wojdylak**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | N/A
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/ApiBundle/Controller/GetAddressLogEntryAction.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+namespace Sylius\Bundle\ApiBundle\Controller;
+
+use Doctrine\Common\Collections\Collection;
+use Sylius\Bundle\ApiBundle\Query\GetAddressLogEntry;
+use Sylius\Component\Addressing\Model\AddressLogEntry;
+use Symfony\Component\Messenger\HandleTrait;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+/** @experimental */
+final class GetAddressLogEntryAction
```
Comment (NoResponseMate) [inline src/Sylius/Bundle/ApiBundle/Controller/GetAddressLogEntryAction.php]:
```suggestion
final class GetAddressLogEntriesAction
```
or
```suggestion
final class GetAddressLogEntryCollectionAction
```

**Review by NoResponseMate — DISMISSED** [review]

File: src/Sylius/Bundle/ApiBundle/Controller/GetAddressLogEntryAction.php (hunk 1 above)
Comment (Wojdylak) [reply src/Sylius/Bundle/ApiBundle/Controller/GetAddressLogEntryAction.php]: `GetAddressLogEntryCollectionAction`

**Review by NoResponseMate — APPROVED** [review]

File: src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Address.xml [hunk 2]
```diff
@@ -61,6 +61,19 @@
                 </attribute>
             </itemOperation>
 
+            <itemOperation name="admin_get_log_entries">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">/admin/addresses/{id}/log-entries</attribute>
+                <attribute name="controller">Sylius\Bundle\ApiBundle\Controller\GetAddressLogEntryCollectionAction</attribute>
```
Comment (jakubtobiasz) [inline src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Address.xml:67]: 🫶🏼

image

**Review by jakubtobiasz — APPROVED** [review]

**Comment by jakubtobiasz** [comment]
Thank you, @Wojdylak!

**Pull request merged**